include/dxc/Support/dxcapi.use.h

```diff
@@ -85,13 +87,13 @@ class DxcDllSupport {
     other.m_createFn2 = nullptr;
   }
 
-  ~DxcDllSupport() { Cleanup(); }
+  virtual ~DxcDllSupport() { Cleanup(); }
 
-  HRESULT Initialize() {
+  HRESULT virtual Initialize() {
     return InitializeInternal(kDxCompilerLib, "DxcCreateInstance");
   }
 
-  HRESULT InitializeForDll(LPCSTR dll, LPCSTR entryPoint) {
+  HRESULT virtual InitializeForDll(LPCSTR dll, LPCSTR entryPoint) {
```

Comment:
I think it should be `InitializeInternal` that's virtual, not `Initialize` or `InitializeForDll`.